modules/local/prepare_for_multiqc.nf

```diff
@@ -0,0 +1,38 @@
+process PREPARE_FOR_MULTIQC {
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pyopenms:3.1.0--py311h9b8898c_0' :
+        'biocontainers/pyopenms:3.1.0--py311h9b8898c_0' }"
+
+    input:
+    tuple val(meta), path(file)
+
+    output:
+    path '*_histogram_mz.tsv'              , emit: hist_mz
+    path '*_histogram_rt.tsv'              , emit: hist_rt
+    path '*_histogram_scores.tsv'          , emit: hist_scores
+    path '*_histogram_xcorr_scores.tsv'    , emit: hist_xcorr
+    path '*_peptide_length.tsv'            , emit: lengths
+    path '*_general_stats.tsv'             , emit: stats
+    path '*_final.tsv'                     , emit: final_tsv
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def digest_mass_range = params.digest_mass_range
```

Comment:
You dont need to define them here, you should be able to directly specify them as python cli args

Reply:
`${params.prec_charge}` e.g.